### PR TITLE
go-ipfs: rename to kubo, update to 0.29.0

### PR DESCRIPTION
**Pull request by Luciogi**
#### Testing the changes
- I tested the changes in this PR: **YES**

#### Local build testing
- I built this PR locally for my native architecture, (x86_64-LIBC)

### CODE REVIEW

File: srcpkgs/kubo/template [hunk 1]
```diff
@@ -30,3 +30,9 @@ post_install() {
 	vdoc CHANGELOG.md
 	vsv ipfs
 }
+
+go-ipfs_package() {
+	build_style=meta
+	depends="kubo-${version}_${revision}"
```
Comment (classabbyamp) [inline srcpkgs/kubo/template:36]:
```suggestion
	depends="kubo>=${version}_${revision}"
```

File: srcpkgs/kubo/template [hunk 2]
```diff
@@ -30,3 +30,9 @@ post_install() {
 	vdoc CHANGELOG.md
 	vsv ipfs
 }
+
+go-ipfs_package() {
+	build_style=meta
+	depends="kubo>=${version}_${revision}"
+	short_desc+=" - Transistional package"
```
Comment (classabbyamp) [inline srcpkgs/kubo/template:37]:
```suggestion
	depends="${sourcepkg}>=${version}_${revision}"
	short_desc+=" - transistional dummy package"
```

Comment (Luciogi) [reply srcpkgs/kubo/template]: Done

File: srcpkgs/kubo/template [hunk 3]
```diff
@@ -1,25 +1,25 @@
...
+changelog="https://github.com/ipfs/kubo/blob/master/CHANGELOG.md"
+distfiles="https://github.com/ipfs/kubo/archive/v${version}.tar.gz"
+checksum=7679ed1b2c6a713b2297ab025d83a50c17fbcebf7e3af3b92be06cb9678a40f5
 
 system_accounts="_ipfs"
 make_dirs="/var/lib/ipfs 0700 _ipfs _ipfs"
 _ipfs_homedir="/var/lib/ipfs"
 
-export GOTOOLCHAIN=go1.20
+#export GOTOOLCHAIN=go1.20
```
Comment (ahesford) [inline srcpkgs/kubo/template:22]: Remove this line.

Comment (Luciogi) [reply srcpkgs/kubo/template]: Done

**Pull request merged**